exercises/yacht/canonical-data.json

```diff
@@ -0,0 +1,261 @@
+{
+    "exercise" : "yacht",
+    "version" : "1.0.0",
+    "comments" : [
+        "The dice are represented always as a list of exactly five integers",
+        "with values beteen 1 and 6 inclusive. The category is an integer.",
+        "the numbers 0 to 5 represent the categories 'ones' to 'sixes'",
+        "Then 6 is full house,",
+        "     7 is four of a kind",
+        "     8 is a little straight 1-5",
+        "     9 is b big straight 2-6",
+        "    10 is Choise, sometimes called Chance",
+        "    11 is Yacht, or five of a kind"
+    ],
+    "cases": [
+	{
+	
+        "description": "Yacht",
+        "property":    "score",
+        "input":       {
+		    "dice":    [5, 5, 5, 5, 5],
```

Comment:
I have a hard time reading this since the column distance between levels is inconsistent. The distance between the opening brace on line 16 and `"description"` on line 18 is only one space, much smaller than the distance between `"input"` and `"dice"`. I would request that the distance between levels be made consistent, for the purpose of readability.

Same request applies to all other cases.